student.rb

```diff
@@ -2,8 +2,10 @@
 class Student < Person
   attr_accessor :classroom
 
-  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
-    super(age, name, parent_permission: parent_permission)
+  # rubocop:disable Style/OptionalBooleanParameter
+  def initialize(_age, classroom, _name = 'Unknown', _parent_permission = true)
+    # rubocop:enable Style/OptionalBooleanParameter    super(age, name, parent_permission: parent_permission)
+    super(age, name, parent_permission)
```

Comment:
- [ ] Please 🙏🏽 correct the name of the variable `_parent_permission` instead of `parent_permission` as defined in the initialize to avoid this issue 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/231789162-1a7dbd27-972e-4c1d-a204-fedffe59018d.png)